server/src/test/java/org/elasticsearch/action/admin/cluster/bootstrap/TransportGetDiscoveredNodesActionTests.java

```diff
@@ -423,8 +422,7 @@ public void testThrowsExceptionIfDuplicateDiscoveredLater() throws InterruptedEx
 
         final GetDiscoveredNodesRequest getDiscoveredNodesRequest = new GetDiscoveredNodesRequest();
         final String ip = localNode.getAddress().getAddress();
-        getDiscoveredNodesRequest.setRequiredNodes(Collections.singletonList(ip));
-        getDiscoveredNodesRequest.setWaitForNodes(2);
+        getDiscoveredNodesRequest.setRequiredNodes(Stream.of(ip, "not-a-node").collect(Collectors.toList()));
```

Comment:
just `Arrays.asList(ip, "not-a-node")`

Reply:
Ok f85df9c4d0b1635f53be92f2dcd0db49c1f2a158